doc/debugger.rst

```diff
@@ -1875,7 +1876,7 @@ GetBytecode request (0x21)
 
 Format::
 
-    REQ <int: 0x21> [<int: level> OR <obj: target> OR <heapptr: target>] EOM
+    REQ <int: 0x21> [<int: level | obj: target | heapptr: target>] EOM
```

Comment:
These notation fixes are a bit inconsistent, couple of them are `<int: something | obj: something>` while others are `<int: something> | <obj: something>`. I'll fix it in a cleanup pull.